### ISSUE TITLE
SearchAbout issue / support

### ISSUE THREAD

**Issue by vsiles** [whonore/Coqtail#10]
Hi !
When I try to evaluate `SearchAbout nat`, the plugin stop responding (it display a couple entries and that's it). Is there a way to use `SearchAbout` with this plugin ?
Note: coq-au-vim has the same issue :p

**Comment by whonore** [comment]
Hi, could you tell me what version of Coq you're using? `SearchAbout` works fine for me on all the versions I've tried.

**Comment by vsiles** [comment]
I'm using Coq from opam: 
```
coqtop -v
The Coq Proof Assistant, version 8.8.1 (October 2018)
compiled on Oct 9 2018 17:40:59 with OCaml 4.02.3
```

Using `SearchAbout nat` in a clean file works, but using it after some `Require` seems to fail, here is an example screen:
![screen shot 2018-10-26 at 16 02 43](https://user-images.githubusercontent.com/7124672/47571389-b7368680-d938-11e8-8403-317fe8c9fd14.png)

**Comment by whonore** [comment]
Ok, it looks like this is a problem for Coq >=8.6 and Python 2. I'll look into it.

**Comment by vsiles** [comment]
It might also have been my fault: I had still traces of `coq-au-vim` around. It seems to be better with these traces deleted. If you have some benchmark you would like me to test, feel free to reach out.

Thanks for your work !

**Comment by whonore** [comment]
I pushed a possible fix to branch [slow-search-fix](https://github.com/whonore/Coqtail/tree/slow-search-fix). When you have a chance could you try pulling it and see if it helps?

**Comment by vsiles** [comment]
It seems ok, and even a bit faster than yesterday ! Thanks a lot. I only tested my problematic use case, but I'll try to test it more by the end of the week.

**Comment by whonore** [comment]
Fixed by 762fdaccc3f569f72a53624682d0f197468f98fc.
